pkg/tnf/handlers/generic/generic.go

```diff
@@ -17,11 +17,15 @@
 package generic
 
 import (
+	"bytes"
 	"encoding/json"
+	"html/template"
```

Comment:
I assume the reason for using HTML templating is because it has escaping, and the input may end up being executed on a command line. I couldn't see anything in the `template` library docs about this use case but it seems sensible.

Reply:
This was originally chosen for security reasons.  However, this may lead to adverse side effects.  I will go ahead and switch to `text/template`.  In reality, our code is executing (sometimes root) commands all over the place, and is not designed to be security software.